lib/entities/entity-types.ts

```diff
@@ -0,0 +1,30 @@
+export type { ApiKey } from './api-key'
```

Comment:
Any reason not to/is it possible to just export * types from each file?

Reply:
I'm picturing scenarios where I need to dig in and use a specific inner type for an entity (e.g. the EntrySys)

Reply:
Wasn't sure about potential clashes between all those files and `export type` didn't give me the asterisk notation. So I thought a sensible selection might make more sense. But you're right, this makes it harder to obtain shapes like `ApiKeyProps`.

Giving it a shot with `export * from` right now. But might make sense to revisit the `index.ts` then as it exports everything under the default export.

@suevalov is there a reason for `entities/index.ts` default export behavior?

Reply:
@zcei That was just convenience on the internal consumption inside the `lib/create-*-api.ts` files. Changing the `lib/entities/index.ts` exports would require some additional adjustments internally but it can be done with relative ease.

Reply:
We'd also need to export all `*Props` types, cause they are actually describing the actual API responses without any additional wrappers on it.

Reply:
From the user point of view, we don't have a big prefrence, or know the distinction of "inner types" / "outer types". 
We mainly want import {Entries, EntryProps} from 'contentful-managment/types' or something like that. If you have groupings, that is also fine => anything is better than the current way.